app/server/models/transfer_account.py

```diff
@@ -194,19 +194,38 @@ def get_approval(self, receiving_address):
                 return approval
         return None
 
-    def approve_and_disburse(self):
-        if not self.is_approved:
-            self.is_approved = True
+    def approve_and_disburse(self, initial_disbursement=None, auto_resolve=False):
+        from server.utils.access_control import AccessControl
 
-            if self.is_beneficiary:
-                disbursement = self.make_initial_disbursement()
+        auto_approve = current_app.config['AUTO_APPROVE_TRANSFER_ACCOUNTS']
+        admin = getattr(g, 'user', None)
+
+        if not self.is_approved and admin:
+            if AccessControl.has_sufficient_tier(admin.roles, 'ADMIN', 'superadmin'):
+                auto_resolve = True
+                self.is_approved = True
+
+            elif AccessControl.has_sufficient_tier(admin.roles, 'ADMIN', 'admin'):
+                auto_resolve = initial_disbursement \
+                               <= current_app.config['MAXIMUM_CUSTOM_INITIAL_DISBURSEMENT']
```

Comment:
spec in comment thread has this as:
> Superadmins must approve non-default amounts.

rather than amounts more than `MAXIMUM_CUSTOM_INITIAL_DISBURSEMENT`, which I think makes sense (or equally, amounts more than default amount, which is also implied from spec)... wondering if we should just deprecate `MAXIMUM_CUSTOM_INITIAL_DISBURSEMENT`

Reply:
Sure, we can deprecate it

Reply:
done